azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1

```diff
@@ -106,14 +106,11 @@ Write-Host
                 write-host "INFO: Fork exists....Proceeding" -ForegroundColor Gray
             }
         }
-    }
-    catch {
-        Write-Host "ERROR: $githubUser/jumpstart-agora-apps Fork doesn't exist, please fork https://github.com/microsoft/jumpstart-agora-apps to proceed....waiting 45 seconds" -ForegroundColor Red
-        start-sleep -Seconds 45
-    }
-} until (
-    $response.full_name -eq "$githubUser/$appsRepo"
-)
+        catch {
```

Comment:
If user hasnt correctly supplied their github username this looks like it will loop forever even if they fork it

Reply:
It waits till the fork is created based on the username they provided, good catch. I will add like 4 retries then break